cmd/vcluster/context/flag_options.go

```diff
@@ -159,7 +163,8 @@ func AddFlags(flags *pflag.FlagSet, options *VirtualClusterOptions) {
 	flags.StringVar(&options.HostMetricsBindAddress, "host-metrics-bind-address", "0", "If set, metrics for the controller manager for the resources managed in the host cluster will be exposed at this address")
 	flags.StringVar(&options.VirtualMetricsBindAddress, "virtual-metrics-bind-address", "0", "If set, metrics for the controller manager for the resources managed in the virtual cluster will be exposed at this address")
 
-	flags.BoolVar(&options.RewriteHostPaths, "rewrite-host-paths", false, "If enabled, syncer will rewite hostpaths in synced pod volumes")
```

Comment:
Can we keep the old flag, but just change the behaviour? Also add a new flag to restore the old behaviour

Reply:
sure, this is moved here now: https://github.com/loft-sh/vcluster/pull/1122/files#diff-c4c21e3be46aa3e2374bdeb86c95b1e5fc10dd95987f14d653c7fb734155ca46R178